.gitignore

```diff
@@ -121,6 +121,7 @@ CMakeCache.txt
 CMakeFiles
 cmake_install.cmake
 install_manifest.txt
+cmake-build-debug
```

Comment:
We should use a more generic pattern than this.

Reply:
open to suggestions

Reply:
The preexisting `build*/` pattern should already work.